installer/templates/phx_web/router.ex

```diff
@@ -6,6 +6,7 @@ defmodule <%= @web_namespace %>.Router do
     plug :fetch_session
     plug :fetch_live_flash
     plug :put_root_layout, html: {<%= @web_namespace %>.Layouts, :root}
+    plug :put_layout, false
```

Comment:
TODO: probably remove after https://github.com/phoenixframework/phoenix/pull/6132 is merged

Reply:
Merged.